### ISSUE TITLE
Sources are wrong

### ISSUE THREAD

**Issue by MsNyxxie** [synclerd/syncler#131]
In BETA: Sources coming in for movies and tv shows are either of the wrong movie/tv show or the wrong season/episode.

**Comment by AltFreq07** [comment]
Yea, non-valid results based on file names dont seem to be removed from the sources list. Even with the remove non-valid results enabled my sources still have a mix of both valid (tick icon) and non-valid (no-tick)

**Comment by royallpotato** [comment]
Yeah, to add to this one further, I've specifically noticed this on shows/movies that are reboots and have similar names. A good example of this is ATLA. The source list for the original animated series is full of sources for the new NF reboot, even when they have a different episode title or say "2024" in the title.

This isn't something I've faced in other apps nearly as often.

**Comment by LoganWolverineX** [comment]
closing GH item.  if needs to be revisited, GH item can be re-opened or new GH item can be created with new details provided. this may be related to v1.x.  no code changes will be done.
